pinot-query-runtime/src/test/java/org/apache/pinot/query/QueryTestSet.java

```diff
@@ -106,6 +106,9 @@ public Object[][] provideTestSql() {
             + " SUM(CASE WHEN a.col2 <> 'foo' AND a.col2 <> 'alice' THEN 1 ELSE 0 END) as unmatch_sum "
             + " FROM a WHERE a.ts >= 1600000000 GROUP BY a.col1"},
 
+        new Object[]{"SELECT a.col1, CASE WHEN sum(a.col3) = 0 THEN 0 ELSE SUM(a.col3) END AS match_sum "
+            + " FROM a WHERE a.ts >= 1600000000 GROUP BY a.col1"},
```

Comment:
Would like to add additional unit tests for V2 as I think this test just verifies Semantics and Syntax, would like to test execution results.

Reply:
Actually we want to add tests for V1. It is already working in V2